diff --git a/Telegram/SourceFiles/data/data_user.cpp b/Telegram/SourceFiles/data/data_user.cpp
index 74fb183..bc6dbb4 100644
--- a/Telegram/SourceFiles/data/data_user.cpp
+++ b/Telegram/SourceFiles/data/data_user.cpp
@@ -609,7 +609,7 @@ bool UserData::isFake() const {
 }
 
 bool UserData::isPremium() const {
-	return flags() & UserDataFlag::Premium;
+	return true;
 }
 
 bool UserData::isBotInlineGeo() const {
diff --git a/Telegram/SourceFiles/main/main_session.cpp b/Telegram/SourceFiles/main/main_session.cpp
index 25324f0..cb6912c 100644
--- a/Telegram/SourceFiles/main/main_session.cpp
+++ b/Telegram/SourceFiles/main/main_session.cpp
@@ -351,11 +351,11 @@ rpl::producer<> Session::downloaderTaskFinished() const {
 }
 
 bool Session::premium() const {
-	return _user->isPremium();
+	return true;
 }
 
 bool Session::premiumPossible() const {
-	return premium() || premiumCanBuy();
+	return true;
 }
 
 bool Session::premiumBadgesShown() const {
@@ -369,7 +369,7 @@ rpl::producer<bool> Session::premiumPossibleValue() const {
 	) | rpl::filter([=](UserData::Flags::Change change) {
 		return (change.diff & UserDataFlag::Premium);
 	}) | rpl::map([=] {
-		return _user->isPremium();
+		return true;
 	});
 	return rpl::combine(
 		std::move(premium),
@@ -378,7 +378,7 @@ rpl::producer<bool> Session::premiumPossibleValue() const {
 }
 
 bool Session::premiumCanBuy() const {
-	return _premiumPossible.current();
+	return true;
 }
 
 bool Session::isTestMode() const {
